libobs/obs.c

```diff
@@ -1890,9 +1890,13 @@ void obs_load_sources(obs_data_array_t *array, obs_load_source_cb cb,
 
 	for (i = 0; i < count; i++) {
 		obs_data_t *source_data = obs_data_array_item(array, i);
-		obs_source_t *source = obs_load_source(source_data);
-
-		da_push_back(sources, &source);
+		if (!obs_get_source_by_name(
```

Comment:
obs_get_source_by_name adds a reference but I don't see a release

Reply:
Good catch. This is some old code from one of the first ways I tried restoring scenes but didn't workout. I'll revert it. This also led me to discover another bug, that was a massive miss on my part